server/events.lua

```diff
@@ -101,7 +97,7 @@ end)
 
 RegisterNetEvent('QBCore:server:OpenServer', function()
     local src = source
-    if QBCore.Functions.HasPermission(src, 'admin') or QBCore.Functions.HasPermission(src, 'god') then
+    if QBCore.Functions.HasPermission(src, 'admin') then
```

Comment:
Requires god as well unless we restructure the permission system, god is considered higher than admin.

Reply:
The HasPermission function allows it already even if you check for another permission that isn't god